Gemfile

```diff
@@ -55,10 +55,10 @@ group :production do
 end
 
 group :development do
-  gem 'capistrano', '3.4.0'
-  gem 'capistrano-bundler', '1.1.4'
+  gem 'capistrano', '3.8.1'
+  gem 'capistrano-bundler', '1.2.0'
   gem 'capistrano-passenger', '0.2.0'
-  gem 'capistrano-rails', '1.1.3'
+  gem 'capistrano-rails', '1.2.3'
   gem 'capistrano-maintenance', '~> 1.0', require: false
```

Comment:
It was locked specifically when I got to it, I think from when this was set up for us by DCE. I wasn't sure whether capistrano was following semantic versioning when I read over the release notes, so I didn't change it.

I can do it now, though.